scripts/BilbyReductionScript.py

```diff
@@ -0,0 +1,337 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+# flake8: noqa
```

Comment:
Is it really necessary to turn off flake8? Wouldn't it make more sense to split up this function into multiple smaller private functions?